bbl_screen-patch/patches/printerui/qml/x1plus/GcodeGenerator.js

```diff
@@ -55,106 +13,48 @@ function createGcode(command, params = {}) {
     }
     return `${gcode}\n`;
 }
-function macros_nozzlecam() {
-    let gcode = G28(0);
-    gcode += G0({x: 240, y: 90, z: 8, accel: 1200});
-    gcode += M960(LEDS.LED_NOZZLE, 1);
-    gcode += M973(OV2740.ON);
-    gcode += M973(OV2740.AUTOEXPOSE);
-    gcode += 'M400 P100 \n';
-    gcode += M973(OV2740.CAPTURE, 1, 1);
-    gcode += M960(LEDS.LED_NOZZLE, 0);
-    gcode += M973(OV2740.OFF);
-    return gcode;
-}
-function macros_tramming(step) {
-    let gcode = M1002({action_code:254,action:0});
-    switch (step) {
-        case TRAMMING_STEP.EXIT:
-            gcode += G1({x: 128, y: 128, z: 1});
-            gcode += M400(0);
-            break;
-        case TRAMMING_STEP.PREPARE:
-            gcode += M17(1.2, 1.2, 0.75);
-            gcode += G90();
-            gcode += M83();
-            gcode += G28(0);
-            gcode += G1({x: 128, y: 128, z: 1});
-            gcode += G292(0);
-            break;
-        case TRAMMING_STEP.REAR_CENTER:
-            gcode += G1({x: 134.8, y: 242.8, z: 0.4, accel: 3600});
-            gcode += M400(0);
-            break;
-        case TRAMMING_STEP.FRONT_LEFT:
-            gcode += G1({x: 33.2, y: 13.2, z: 0.4, accel: 3600});
-            gcode += M400(0);
-            break;
-        case TRAMMING_STEP.FRONT_RIGHT:
-            gcode += G1({x: 222.8, y: 13.2, z: 0.4, accel: 3600});
-            gcode += M400(0);
-            break;
-    }
-    gcode += M1002({action_code:1,action:0});
-    return gcode;
-}
-function macros_vibrationCompensation(f1,f3,nozzleTemp,bedTemp) {
-    let f2 = Math.floor((f3-f1)*0.5); //midpoint
-    
-    let header = M1002({action_code:13,action:0});
-    let footer = M1002({action_code:0,action:0});
-    if (nozzleTemp>0 && nozzleTemp <= 300){
-        header += M109(nozzleTemp);
-        footer += M109(0);
-    }
-    if (bedTemp>0 && bedTemp <= 300){
-        header += M140(bedTemp,true);
-        footer += zzzM140(0,false);
-    }
 
```

Comment:
In the old mechanism, we M1002(13) *before* we M140(bedTemp).  This is, at least, a behavioral change in the new one; is it intentional?